alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportDeleteWorkflowAction.kt

```diff
@@ -270,7 +264,7 @@ class TransportDeleteWorkflowAction @Inject constructor(
                 }
                 workflow.copy(id = hit.id, version = hit.version)
             }
-            val workflowMonitors = workflows.filter { it.id != workflowIdToBeDeleted }.flatMap { (it.inputs[0] as CompositeInput).getMonitorIds() }.distinct()
+            val workflowMonitors = workflows.flatMap { (it.inputs[0] as CompositeInput).getMonitorIds() }.distinct()
```

Comment:
Why are we removing this filter? We need to ignore the workflow we want to delete.